opencolorio_config_aces/config/cg/generate/resources/OpenColorIO-Config-ACES _CG_ Transforms - CG Config - Mapping.csv

```diff
@@ -1,20 +1,20 @@
 Ordering,Transform Name,ACEStransformID,CLFtransformID,Linked DisplayColorSpace Style,Interface,Encoding,Categories
 100,ACES - ACEScc,urn:ampas:aces:transformId:v1.5:ACEScsc.Academy.ACEScc_to_ACES.a1.0.3,,,ColorSpace,log,file-io
-100,ACES - ACEScct,urn:ampas:aces:transformId:v1.5:ACEScsc.Academy.ACEScct_to_ACES.a1.0.3,,,ColorSpace,log,"file-io,working-space"
+100,ACES - ACEScct,urn:ampas:aces:transformId:v1.5:ACEScsc.Academy.ACEScct_to_ACES.a1.0.3,,,ColorSpace,log,file-io
 100,ACES - ACEScg,urn:ampas:aces:transformId:v1.5:ACEScsc.Academy.ACEScg_to_ACES.a1.0.3,,,ColorSpace,scene-linear,"file-io,working-space"
-100,Input - Generic - sRGB - Texture,,urn:aswf:ocio:transformId:v1.0:ACES.OCIO.AP0_to_Rec709-sRGB.c1.v1,,ColorSpace,sdr-video,file-io
+100,Input - Generic - sRGB - Texture,,urn:aswf:ocio:transformId:1.0:OCIO:Utility:AP0_to_Rec709-sRGB:1.0,,ColorSpace,sdr-video,file-io
 100,Output - P3-D65 ST2084 (1000 nits),urn:ampas:aces:transformId:v1.5:RRTODT.Academy.P3D65_1000nits_15nits_ST2084.a1.1.0,,DISPLAY - CIE-XYZ-D65_to_ST2084-P3-D65,ViewTransform,hdr-video,file-io
```

Comment:
In the Gist of the config, the ST2084-P3-D65 colorspace has an encoding of sdr-video, which is wrong.  Although here it says hdr-video, which is correct.  So maybe the Gist is out of date?